tests/consumers/duckdb_consumer.py

```diff
@@ -0,0 +1,64 @@
+import string
+from collections.abc import Iterable
```

Comment:
If this is for testing only, I think the convention is to grab it from the `typing` module (although it doesn't make a difference in application)